.setup/dev-upgrade-h2.sh

```diff
@@ -119,7 +121,11 @@ update_apache() {
 
     info "Double check that HTTP/2 module is enabled"
     phpver=$(php -r 'print PHP_MAJOR_VERSION.".".PHP_MINOR_VERSION;')
+    #TODO: REMOVE THIS https://github.com/Submitty/Submitty/actions/runs/11983433520/job/33412925940#step:22:50
```

Comment:
This actions result will be deleted after a month or two.  It would be better to link to an issue and paste as much of the relevant bits of the log as possible in the issue.